packages/vscode-extension/src/officeChat/commands/create/officeCreateCommandHandler.ts

```diff
@@ -70,27 +70,34 @@ export default async function officeCreateCommandHandler(
         ),
       ];
       officeChatTelemetryData.chatMessages.push(...describeProjectChatMessages);
-
       await verbatimCopilotInteraction(
         "copilot-gpt-3.5-turbo",
         describeProjectChatMessages,
         response,
         token
       );
+
       if (matchedResult.type === "sample") {
-        const folder = await showOfficeSampleFileTree(matchedResult, response);
+        const sampleInfos = await showOfficeSampleFileTree(matchedResult, response);
+        const folder = (sampleInfos as any)?.["path"];
+        const hostType = (sampleInfos as any)?.["host"].toLowerCase();
         const sampleTitle = localize("teamstoolkit.chatParticipants.create.sample");
+        officeChatTelemetryData.setHostType(hostType);
+        const matchResultInfo = "sample";
```

Comment:
This has been defined in `matchedResult.type`, remove this variable.

Reply:
will fix, thanks